server/app/controllers/admin/AdminImportController.java

```diff
@@ -99,4 +102,18 @@ public Result hxImportProgram(Http.Request request) {
     }
     return ok(adminImportViewPartial.renderProgramData(programMigrationWrapper).render());
   }
+
+  // TODO(#7087) remove this when we add the ability to parse questions into QuestionDefinitions
+  private String trimQuestionsOffJson(String jsonString) {
+    int indexOfQuestions = jsonString.indexOf("\"questions\"");
```

Comment:
*issue*: I think this will break if the word `"questions"` is anywhere in a program name or description, right?

Reply:
Yeah it will. But this will never be usable by admin. It's just a place holder so I can get tests to pass and continue manual testing while I move on to the next thing.